packages/manager/modules/anthos/src/anthos.routing.js

```diff
@@ -8,13 +8,22 @@ export default /* @ngInject */ ($stateProvider) => {
       transition
         .injector()
         .getAsync('tenants')
-        .then((tenants) =>
-          tenants.length === 0
-            ? {
-                state: 'anthos.onboarding',
-              }
-            : false,
-        ),
+        .then((tenants) => {
+          if (tenants.length === 0) {
+            return {
+              state: 'anthos.onboarding',
+            };
+          }
+          if (tenants.length === 1) {
```

Comment:
It should be an `else` or a fall through here, no need to check for length twice. If you are comfortable go with `tenants.length > 0`

Reply:
I am not sure to get your point of view.
The idea is :
customer target list page if we have 
   0 tenants => go to onboarding
   1 tenants => go to details page
   > 1 => can display list of tenants

Reply:
@varun257 let me know if you need more explantion